order.go

```diff
@@ -88,6 +98,17 @@ func (c *Client) Order(i *Instrument, o OrderOpts) (*OrderOutput, error) {
 		a.Trigger = "stop"
 	}
 
+	// for now we only support Percentage
+	if strings.ToLower(o.TrailingPeg.Type) == "percentage" {
+		a.TrailingPeg.Type = "percentage"
+		a.TrailingPeg.Percentage = o.TrailingPeg.Percentage
+
+		a.StopPrice = o.StopPrice
+		a.Trigger = "stop"
+
+		a.ExtendedHours = false // Trailing stop not for extended hours
```

Comment:
should we remove this line?

Reply:
yes, by default `bool` variables are `false`

Reply:
the issue is that now in our mew, we default this to true, so if I remove this line, we need to update the stop trailing command to default to false, which I think is the right thing to do